Lib/test/test_dbm.py

```diff
@@ -172,6 +176,10 @@ def test_whichdb_ndbm(self):
         with open(db_file, 'w'):
             self.addCleanup(os_helper.unlink, db_file)
         self.assertIsNone(self.dbm.whichdb(db_file[:-3]))
+        path = os_helper.FakePath(db_file)
+        with open(path, 'w'):
+            self.addCleanup(os_helper.unlink, path)
```

Comment:
It is not needed. It creates an empty file and makes it removed after the test, but it is already done above.